### ISSUE TITLE
google sdk c++ is not supporting wake-word engine.

### ISSUE THREAD

**Issue by Harman-MagicBox** [googlesamples/assistant-sdk-python#364]
Hi all,

when I run this command, 

./run_assistant_audio --credentials ./credentials.json

It always active i.e. without even saying the "Ok Google" wake word.

Do let me know if Google sdk C++ support "wake-word" or not.

**Comment by Fleker** [comment]
The C++ version does not support wake word detection.

**Comment by Harman-MagicBox** [comment]
Thanks :)
Any other version of Programming, apart from python support wake word detection?

**Comment by Fleker** [comment]
No, there is just the Python library which supports it. The other samples use the gRPC API, which does not have any local wake word component.

**Comment by swat1998** [comment]
Is the python library for wake word still working? Cannot find it though.....
